itest/client_harness.go

```diff
@@ -119,6 +119,9 @@ func (c *clientHarness) start() error {
 }
 
 func (c *clientHarness) cleanup() error {
-	c.cancel()
+	// we always want to cancel the context, even if
```

Comment:
s/we/We

Also - this change is more about canceling the context after closing the conn. Not about closing it regardless of error.

Reply:
Thanks, updated this message :)!